### PR TITLE
[IMP] accounting: internal transfers

### PR DESCRIPTION
**Pull request by chiaraprattico**
taskid-3663508

### CODE REVIEW

File: content/applications/finance/accounting/payments/internal_transfers.rst [hunk 1]
```diff
@@ -2,45 +2,37 @@
...
 Log an internal transfer
 ------------------------
 
-From the Accounting Dashboard, click on the drop-down selection button (:guilabel:`⋮`) on one of
-your banks. In the :guilabel:`New` column click on :guilabel:`Internal Transfer` and enter the
-information related to the transfer.
+In order to transfer 1000 USD from Bank A to Bank B, from the Accounting Dashboard, click on the
+drop-down selection button (:guilabel:`⋮`) on one of your banks, then click on :guilabel:`Payments`
+and each payment can be marked as :guilabel:`Internal Transfer`. A :guilabel:`Destination Journal`
+must be selected in order to :guilabel:`Confirm` the internal transfer.
```
Comment (toaa-odoo) [inline content/applications/finance/accounting/payments/internal_transfers.rst:28]:
```suggestion
In order to transfer 1000 USD from Bank A to Bank B, from the Accounting Dashboard, click on the
drop-down selection button (:guilabel:`⋮`) on one of your banks, then click on :guilabel:`Payments`
and each payment can be marked as :guilabel:`Internal Transfer`. A :guilabel:`Destination Journal`
must be selected in order to :guilabel:`Confirm` the internal transfer.
```
If you want to transfer 1000 USD from bank A to bank B, for example, from the Accounting dashboard, click the **drop-down** selection button (:guilabel:`⋮`) on one of your bank journals, and click :guilabel:`Payments`. Each payment can be marked as :guilabel:`Internal Transfer`, however, a :guilabel:`Destination Journal` must be selected in order to :guilabel:`Confirm` the internal transfer.

.. note::
   You can only make **internal transfers** on *unposted* payments.

**Review by toaa-odoo — CHANGES_REQUESTED** [review]
Hey @chiaraprattico, thanks for your work! I made a small suggestion you may want to consider, and also there is a grammar mistake at line 72 (I cannot comment on it because it wasn't modified): There *is* one [...] :)

**Review by toaa-odoo — APPROVED** [review]

File: content/applications/finance/accounting/payments/internal_transfers.rst [hunk 2]
```diff
@@ -2,45 +2,40 @@
...
+internal transfers.
+
+.. seealso::
+   :doc:`How to add an additional bank account <../bank>`
 
 Configuration
 =============
 
 An internal transfer account is automatically created on your database based on your company's
 localization and depending on your country’s legislation. If needed, the default :guilabel:`Internal
```
Comment (auva-odoo) [inline content/applications/finance/accounting/payments/internal_transfers.rst:15]:
```suggestion
To modify the default :guilabel:`Internal
```

File: content/applications/finance/accounting/payments/internal_transfers.rst [hunk 3]
```diff
@@ -2,45 +2,40 @@
...
-transfer account` can be modified by going to :menuselection:`Accounting app --> Configuration -->
-Settings` and then under the :guilabel:`Default Accounts` section.
-
-.. note::
-   At least two bank accounts are needed to make internal transfers. Refer to
-   :doc:`Bank and cash accounts section <../bank>` to see how to add an additional bank
-   account to your database.
+transfer account` can be modified by going to :menuselection:`Accounting --> Configuration -->
+Settings --> Default Accounts section`.
```
Comment (auva-odoo) [inline content/applications/finance/accounting/payments/internal_transfers.rst:17]:
```suggestion
transfer account`, go to :menuselection:`Accounting --> Configuration --> Settings`, then scroll down to the :guilabel:`Default Accounts` section.
```

File: content/applications/finance/accounting/payments/internal_transfers.rst [hunk 4]
```diff
@@ -2,45 +2,40 @@
...
 
 Register an internal transfer from one bank to another
 ======================================================
 
-Let's say you have two bank accounts registered on your database and you want to transfer 1,000 USD
-from Bank A to Bank B.
-
 Log an internal transfer
```
Comment (auva-odoo) [inline content/applications/finance/accounting/payments/internal_transfers.rst:22]: See my comment on the reconciliation section, but if you remove the reconciliation section of this doc, I would remove this sub-title as well as it would not be necessary.

File: content/applications/finance/accounting/payments/internal_transfers.rst [hunk 5]
```diff
@@ -2,45 +2,40 @@
...
-your banks. In the :guilabel:`New` column click on :guilabel:`Internal Transfer` and enter the
-information related to the transfer.
-
-.. image:: internal_transfers/internal_transfer.png
-   :align: center
-   :alt: Fill in the information related to your internal transfer
+If you want to transfer 1000 USD from Bank A to Bank B, from the Accounting Dashboard, click on the
+drop-down selection button (:guilabel:`⋮`) on one of your banks, then click on :guilabel:`Payments`.
+Each payment can be marked as :guilabel:`Internal Transfer`, however, a :guilabel:`Destination Journal`
+must be selected in order to :guilabel:`Confirm` the internal transfer.
```
Comment (auva-odoo) [inline content/applications/finance/accounting/payments/internal_transfers.rst:28]: I would rephrase this to a) remove the example part of the explanation as i don't feel it's necessary b) phrase the second part as more of a procedure:

```suggestion
If you want to transfer money from one bank to another, access the Accounting Dashboard, click the drop-down selection button (:guilabel:`⋮`) on the bank from which you want to make the transfer, then click guilabel:`Payments`. Select or create a payment, tick the :guilabel:`Internal Transfer` checkbox, and select a :guilabel:`Destination Journal` before you :guilabel:`Confirm` the internal transfer.
```

File: content/applications/finance/accounting/payments/internal_transfers.rst [hunk 6]
```diff
@@ -2,45 +2,40 @@
...
 
 .. note::
-   Fill in the :guilabel:`Memo` field for automatic reconciliation.
+  You can only make **internal transfers** on *unposted* payments.
+
+.. image:: internal_transfers/transfer-check.png
+   :align: center
```
Comment (auva-odoo) [inline content/applications/finance/accounting/payments/internal_transfers.rst:34]: remove this

File: content/applications/finance/accounting/payments/internal_transfers.rst [hunk 7]
```diff
@@ -2,45 +2,40 @@
...
+   :alt: Internal transfer activation
 
-:guilabel:`Save` and :guilabel:`Confirm` to register your internal transfer. The money is now booked
-in the transfer account and another payment is **automatically** created in the destination journal
-(Bank B).
+The money is now booked in the transfer account and another payment is automatically created in the
+destination journal.
 
 Bank journal (Bank A)
```
Comment (auva-odoo) [inline content/applications/finance/accounting/payments/internal_transfers.rst:40]: I would put the tables in an example box.

Also, regarding the second title:
- Is the "Automated booking" part really necessary?
- it's a bit of a detail but I would remove the capital letters in "Bank B"

File: content/applications/finance/accounting/payments/internal_transfers.rst [hunk 8]
```diff
@@ -77,38 +72,30 @@ Automated booking - Bank journal (BANK B)
      - **$1,000**
 
 .. note::
-   There are one outstanding payment and one outstanding receipt pending in your two bank account
+   There is one outstanding payment and one outstanding receipt pending in your two bank account
```
Comment (auva-odoo) [inline content/applications/finance/accounting/payments/internal_transfers.rst:75]: If you do decide to have the tables in an example box, I'd remove the note and include the text in the example box after the tables.

File: content/applications/finance/accounting/payments/internal_transfers.rst [hunk 9]
```diff
@@ -77,38 +72,30 @@ Automated booking - Bank journal (BANK B)
...
    journals, because the bank statement confirming the sending and receiving of the money has not
    been booked yet.
 
-.. image:: internal_transfers/outstanding-payments-receipts.png
-   :align: center
-   :alt: Outstanding Payments/Receipts pending bank statement booking
+Internal transfer reconciliation
```
Comment (auva-odoo) [inline content/applications/finance/accounting/payments/internal_transfers.rst:79]: Is this whole section necessary? Is there anything in the reconciliation process that's specific to internal transfers? 
If not, I would maybe just add a sentence at the end of the previous section explaining that reconciliation is done as usual + keep the seealso linking to the reconciliation doc page.

**Review by auva-odoo — CHANGES_REQUESTED** [review]
Hi @chiaraprattico, thank you for your work! 
@toaa-odoo, tagging you in case you want to take care of this while Chiara is on holiday :) Sorry, I decided not to force-push the changes since I have some more questions/suggestions. 
Don't hesistate to let me know if you want to discuss this!

**Review by auva-odoo — APPROVED** [review]
@toaa-odoo thanks for the changes, I pushed a few more:
- typos/missing words
- removed the image files since none of them are used anymore
- removed the h3 title since it isn't necessary anymore
- removed the last line in the example box since it applies to the reconciliation process (at least from what I understood, please correct me if I'm wrong!)
- move the sentence on reconciliation after the example box since the example box doesn't relate to reconciliation; I know it's not ideal to have just one sentence in between 2 boxes but I think it'll be fine :)

Feel free to revert any changes you don't agree with. r+ whenever you're ready :)
@robodoo delegate=toaa-odoo

**Comment by toaa-odoo** [comment]
@robodoo r+
